src/checker.py

```diff
@@ -68,6 +81,11 @@ def _get_finish_args_extra_data_from_json(self, json_data):
     def _get_module_data_from_json(self, json_data):
         external_data = []
         for module in json_data.get('modules', []):
+            if isinstance(module, str):
+                module_path = os.path.join(os.path.dirname(self._manifest),
+                                           module)
```

Comment:
@ramcq for this to work, the `check-flatpak-external-apps` job will have to be modified to recursively clone each repo, rather than just running `wget` on each URL.